<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>595</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>808</width>
      <height>605</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="new_expense_stack">
     <widget class="QLabel" name="amount_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="description_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="country_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="country_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>510</y>
        <width>179</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="category_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Categories</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget1">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>200</y>
        <width>241</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="description_input">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modify_expense_stack">
     <widget class="QLineEdit" name="country_input_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="description_label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
     <widget class="QLabel" name="country_label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="amount_input_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="category_label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Categories</string>
      </property>
     </widget>
     <widget class="QLabel" name="amount_label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="description_input_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>241</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="expense_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Expense</string>
      </property>
     </widget>
     <widget class="QListWidget" name="expenseWidget2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>510</y>
        <width>179</width>
        <height>32</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="ID_label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="ID_label_2_desc">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>480</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="show_expenses">
     <widget class="QTableWidget" name="expenseTableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>421</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>421</height>
       </size>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QTableWidget" name="CategoryTableWidget">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>301</width>
        <height>421</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="total_label">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="total_label_number">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="total_label_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Category breakdown</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="categoryTableWidget">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>281</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>471</width>
        <height>421</height>
       </size>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="categories_label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Categories</string>
      </property>
     </widget>
     <widget class="QListWidget" name="categorieslistWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_button">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>300</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modify_button">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>300</y>
        <width>94</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_button">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>420</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_categories_line">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>261</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Expense"/>
    <addaction name="actionModify_an_Expense"/>
    <addaction name="actionShow_all_Expense"/>
    <addaction name="separator"/>
    <addaction name="actionCategories"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Expense">
   <property name="text">
    <string>New Expense</string>
   </property>
  </action>
  <action name="actionModify_an_Expense">
   <property name="text">
    <string>Modify an Expense</string>
   </property>
  </action>
  <action name="actionShow_all_Expense">
   <property name="text">
    <string>Show all Expense</string>
   </property>
  </action>
  <action name="actionAdd_a_Category">
   <property name="text">
    <string>Add a Category</string>
   </property>
  </action>
  <action name="actionCategories">
   <property name="text">
    <string>Categories</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>amount_input</tabstop>
  <tabstop>description_input</tabstop>
  <tabstop>country_input</tabstop>
  <tabstop>listWidget1</tabstop>
  <tabstop>expenseWidget2</tabstop>
  <tabstop>amount_input_2</tabstop>
  <tabstop>description_input_2</tabstop>
  <tabstop>country_input_2</tabstop>
  <tabstop>listWidget2</tabstop>
  <tabstop>expenseTableWidget</tabstop>
  <tabstop>CategoryTableWidget</tabstop>
  <tabstop>categoryTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
